src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -614,7 +614,7 @@ public virtual NamedTypeSymbol TupleUnderlyingType
         /// <remarks>
         /// See Type::computeManagedType.
         /// </remarks>
-        internal abstract bool IsManagedType { get; }
+        internal abstract bool IsManagedType(ConsList<FieldSymbol> fieldsBeingBound = null);
```

Comment:
Do we need to allow `null`? Is the behavior different for `null` rather than `ConsList<FieldSymbol>.Empty`?